src/main/java/Duke.java

```diff
@@ -1,10 +1,90 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+//todo: modularise code
```

Comment:
Consider using a javadoc command to describe the method. You can find out more here: https://www.baeldung.com/javadoc.